css-cascade-7/Overview.bs

```diff
@@ -0,0 +1,402 @@
+	It is currently an Exploratory Working Draft:
+	if you are implementing anything, please use Level 6 as a reference.
+	We will merge the Level 6 text into this draft once it reaches CR.
+
+<!-- Big Text: Stylesheet 
+
+ ███▌  █████▌ █   ▐▌ █▌    █████▌  ███▌  █▌  █▌ █████▌ █████▌ █████▌
+█▌  █▌   █▌   ▐▌  █  █▌    █▌     █▌  █▌ █▌  █▌ █▌     █▌       █▌  
+█▌       █▌    █ ▐▌  █▌    █▌     █▌     █▌  █▌ █▌     █▌       █▌  
+ ███▌    █▌    ▐▌█   █▌    ████    ███▌  █████▌ ████   ████     █▌  
+    █▌   █▌     █▌   █▌    █▌         █▌ █▌  █▌ █▌     █▌       █▌  
+█▌  █▌   █▌     █▌   █▌    █▌     █▌  █▌ █▌  █▌ █▌     █▌       █▌  
+ ███▌    █▌     █▌   █████ █████▌  ███▌  █▌  █▌ █████▌ █████▌   █▌  
+-->
+
+<h2 id='stylesheet-section'>
+Stylesheet</h2>
+
+	A <dfn export>stylesheet</dfn>
+	is a collection of style rules. 
+	
+	These collections may exist as:
+	<ol>
+		<li>An instance of file with the <a href="https://www.w3.org/TR/css-cascade-5/#content-type">CSS Style Sheet Content-Type</a>, embedded within a document.
+		<li>A collection of CSS rules within a document.
+		<li><a href="https://www.w3.org/TR/cssom/#the-stylesheet-interface">CSSOM Stylesheet</a> objects.
+		<li>Any of the above may be subdivided into <dfn>named stylesheet</dfn> with the ''@sheet'' rule.
+	</ol>
+
+	A <dfn>sheet-name</dfn> is a string identifying
+	 inner named sheets within a stylesheet.
+
+	Syntactically, a stylesheet must include an explicit [=sheet-name=], and is represented
+	by the <<sheet-name>> in ''@sheet'' and ''@import'' rules,
+	which is a single <<ident>> token:
+
+	<pre class='prod'>
+		<dfn><<sheet-name>></dfn> = <<ident>>
+	</pre>
+
+<h3 id="at-ruledef-sheet">
+Named Stylesheets: the `@sheet` block at-rule</h3>
+
+	The ''@sheet'' [=block at-rule=]
+	assigns its style rules to a particular [=named stylesheet=].
+	This sheet-assignment syntax is:
+
+	<pre class='prod'>
+		@sheet <<sheet-name>> {
+			<<rule-list>>
+		}
+	</pre>
+
+	Such ''@sheet'' block rules have the same restrictions and processing
+	as a [=conditional group rule=] [[CSS-CONDITIONAL-3]]
+	with a false condition.
+
+Note: This means that rules within an ''@sheet'' block are not applied 
+until their [=named stylesheet=] is explicitly embedded into a document.
+
+	<div class=example>
+		<pre class=lang-css>
+		&lt;style>
+		@sheet framework {
+			h1 { color: maroon; }
+		}
+		&lt;/style>
+		&lt;h1>Sample text&lt;/h1>
+		</pre>
+		In this example, the ''h1'' tag does not have the maroon color applied, as the ''@sheet'' 
+		named "framework" was not explicitly embedded into the document.
+	</div>
+
+	''@sheet'' [=block at-rules=] may not be nested within any other CSS rules.
+
+<h3 id="sheet-declaration">
+Declaring Named Stylesheets</h3>
+
+	A [=named stylesheet=] may be declared by using
+	an [[#at-ruledef-sheet|@sheet block at-rule]],
+	assigning its child style rules into that sheet.
```

Comment:
I think this paragraph is a better candidate for the `<dfn>` of "named stylesheet" than where it is currently, and that this paragraph says mostly the same thing as the paragraph on lines 93-95. I'd suggest moving this text up to that location, editing down as needed, and making this instance of "named stylesheet" the `<dfn>` for that term.